trulens_eval/examples/experimental/appui_example.ipynb

```diff
@@ -42,6 +42,13 @@
     ")"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.    aui.d
</pre><p>Why do we want to call this with an attribute simply the letter <code>d</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Renamed to "widget" instead.